autoAdjustHwloc.hpp

```diff
@@ -149,26 +149,25 @@ class autoAdjust
 					/* if more hashes available than objects in the current depth of the topology
 					 * than divide with round down else round up
 					 */
-					int extraHash = numHashL3 > numL2 ? numHashL3 / numL2 : (numHashL3 + numL2 - 1) / numL2;
+					size_t extraHash = numHashL3 > numL2 ? numHashL3 / numL2 : (numHashL3 + numL2 - 1) / numL2;
 					numHashL3 -= extraHash;
 					if (numHashL3 < 0)
 						numHashL3 = 0;
 					numHashes += extraHash;
 					//add L2 hashes
 					numHashes += ( l2Cache + m_scratchPadMemSize / 2llu ) / m_scratchPadMemSize;
-					int numCachesLeft = numL2;
```

Comment:
We must keep this, this variable because they is reduced during moving over the system component tree in `getConfig`